packages/web-react/src/components/Box/useBoxStyleProps.ts

```diff
@@ -14,17 +15,21 @@ export const useBoxStyleProps = (
   props: Partial<SpiritBoxProps<ElementType>>,
 ): UseBoxStyleProps<Partial<SpiritBoxProps<ElementType>>> => {
   const { backgroundColor, borderColor, borderRadius, borderStyle, borderWidth, ...restProps } = props || {};
-  const boxBackgroundColor = backgroundColor ? `bg-${backgroundColor}` : '';
-  let boxBorderColor = borderColor ? borderColor.replace('', 'border-') : '';
+  const boxBackgroundClassName = useClassNamePrefix(`bg-${backgroundColor}`);
+  const boxBorderClassName = useClassNamePrefix('border-');
+  const boxRadiiClassName = useClassNamePrefix('rounded-');
```

Comment:
```suggestion
  const boxRadiusClassName = useClassNamePrefix('rounded-');
```
?

borders and backgrounds class names are not plural :)